app/controllers/concerns/string_redacter.rb

```diff
@@ -0,0 +1,7 @@
+module StringRedacter
+  extend ActiveSupport::Concern
```

Comment:
```suggestion
  module_function
```

Personally I'd go with a static method so we'd reference this directly like `StringRedacter.redact_alphanumeric('abcdef')` instead of just `redact_alphanumeric`, one of the advantages is that it's easier to see where this comes from at call sites.

Reply:
Done 70c79e6673d90e1cd56d459ea4375535e62039aa and moved to services 7f3a351beea39fdb2ea30e40f49c2d5ffee606c8